tailwind.config.js

```diff
@@ -30,7 +30,8 @@ module.exports = {
         "2xs": "5rem",
       },
       boxShadow: {
-        xl: "-15px -15px #7EAC8A",
+        xl: "-40px -35px #7EAC8A",
```

Comment:
changing xl values will cause problems in other components, please don't override existing values, instead create a new one with a different name, xxl for example. 
Component using xl: cart-item.